packages/firestore/src/protos/update.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+
+# Variables
+PROTOS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+WORK_DIR=`mktemp -d`
+
+# check if tmp dir was created
+if [[ ! "$WORK_DIR" || ! -d "$WORK_DIR" ]]; then
```

Comment:
What are you intending to guard against with `! "$WORK_DIR"`? As long as you're quoting `$WORK_DIR` just checking that it's a directory should be enough here.

Note that if `mktemp` fails it will return non-zero status and that will fail the script due to `set -e`.

Reply:
I copypasta'd this from elsewhere.  I just removed the check entirely given your point about `set -e` and the fact that this isn't mission critical.